### ISSUE TITLE
Playlists with >100 videos aren't being displayed on Invidio.us web app.

### ISSUE THREAD

**Issue by rsapkf** [iv-org/invidious#1260]
Playlists with >100 videos aren't being displayed on Invidio.us web app.
For example:
https://invidio.us/playlist?list=PL-osiE80TeTt2d9bfVyTiXJA-UTHn6WwU (142 videos)

When I query the Invidious API for playlists with more than 100 videos, I get this response:
`GET: https://invidio.us/api/v1/playlists/PL-osiE80TeTt2d9bfVyTiXJA-UTHn6WwU`

```
{
  "error": "Expected Hash for #[]?(key : String), not Array(JSON::Any)"
}
```

Also,
playlists of a channel aren't being displayed:
https://invidio.us/channel/UCCezIgC97PvUuR4_gbFUs5g/playlists

**Comment by gloschtla** [comment]
For the latter problem, I also opened an issue #1255 

It's true: smaller playlist are still loaded. Just realized it.

**Comment by nachanon** [comment]
This is a recent occurrence; probably due to changes from YouTube's side.

### Compare and Contrast ###

- Long playlist (400+ entries): <https://invidio.us/playlist?list=PLRS4T4F2sF1p4JAnoS60z-9IVwkCic_IT> (You would see that **videos list is missing**)
- Short playlist (sub-50 entries): <https://invidio.us/playlist?list=PLRS4T4F2sF1qnv9FQQaVd9U8jZSFj8EKw> (The videos list got displayed normally)

Browser: Pale Moon 25.8.1 (32-bit GNU/Linux), with **JavaScript disabled**

**Comment by SuperSandro2000** [comment]
@nachanon first example does not work in a good browser with js either.

**Comment by omarroth** [comment]
Should be fixed with 8c65b8c16fe06375b7b8c29514a065626a12fe04.

**Comment by rsapkf** [comment]
The problem seems to be fixed now. But the API call only returns the first 100 videos in the response. I am developing an [extension](https://github.com/rsapkf/invidious-playlist-duration) to get the duration of the entire playlist that uses the Invidious API but now the result for playlists with more than 100 videos is inaccurate because it only totals the first 100 videos.

**Comment by gloschtla** [comment]
Will there be an upgraded release? 

> Should be fixed with [8c65b8c](https://github.com/omarroth/invidious/commit/8c65b8c16fe06375b7b8c29514a065626a12fe04).
